frontend/package.json

```diff
@@ -39,8 +39,8 @@
     "validate-map-style": "node src/Map/style/scripts/validate.js",
     "build-map-style": "yarn build-map-icons; node src/Map/style/scripts/build.js",
     "build-map-icons": "spritezero public/mapboxstyle/sprite src/Map/style/icons/; spritezero --retina public/mapboxstyle/sprite@2x src/Map/style/icons/",
-    "build-css": "node-sass-chokidar src/ -o src/",
-    "watch-css": "yarn run build-css && node-sass-chokidar src/ -o src/ --watch --recursive",
+    "build-css": "node-sass-chokidar --include-path ./src src/ -o src/",
+    "watch-css": "yarn run build-css && node-sass-chokidar --include-path ./src src/ -o src/ --watch --recursive",
```

Comment:
@briaguya, Hey you cool with this? @ifsimicoded figured out a way to remove all the ugly relative paths for sass. She's looking into something similar for js. She wants to make sure this doesn't raise any red flags for you.

Reply:
I like it, no red flags here